tslint.json

```diff
@@ -6,6 +6,11 @@
     "tslint-config-prettier",
     "tslint-plugin-prettier"
   ],
+  "linterOptions": {
+    "exclude": [
+      "./scripts/*"
```

Comment:
Let's make this scripts/templates/* 👍

Reply:
Especially now we will be adding js scripts.